### PR TITLE
rustup to current master

### PR DESCRIPTION
**Pull request by globin**

### CODE REVIEW

File: openssl-sys/build.rs [hunk 1]
```diff
@@ -7,10 +7,10 @@ use std::env;
 use std::default::Default;
 
 fn main() {
-    let target = env::var_string("TARGET").unwrap();
+    let target = env::var("TARGET").unwrap();
 
     if target.contains("android") {
-        let path = env::var_string("OPENSSL_PATH").ok()
+        let path = env::var("OPENSSL_PATH").ok()
```
Comment (sfackler) [inline openssl-sys/build.rs]: I think this needs to continue being var_string or it won't print properly below.

Comment (sfackler) [inline openssl-sys/build.rs]: Oops, nevermind.

**Comment by sfackler** [comment]
Thanks! I'll merge this when the nightly lands (it looks like the mac builder broke last night)

**Comment by alexcrichton** [comment]
(nightlies have landed)

**Comment by bombless** [comment]
Just out of curiosity, why that `Unique` thing [seems landed 3 weeks ago](https://github.com/rust-lang/rust/commit/f90c3864b66ba28c5cda46a32a564d77b0c0f848) ..

**Comment by sfackler** [comment]
@bombless the commit was authored 22 days ago but it didn't merge until yesterday.

**Pull request merged**